### PR TITLE
[Ready for Review] Update BinderHub chapter

### PR DESCRIPTION
**Pull request by sgibson91**
### Summary

Some minor updates to the BinderHub chapter. This is mainly for Hacktoberfest :wink:

### List of changes proposed in this PR (pull-request)

* Swap out old BinderHub image for Scriberia one
* Fix typos and tweak language

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/content/binderhub/binderhub.md [hunk 1]
```diff
@@ -135,28 +135,28 @@ At minimum, you'll need to be able to assign [Role Based Access Control (RBAC)](
...
+- [Zero-to-JupyterHub with Kubernetes documentation](https://zero-to-jupyterhub.readthedocs.io/en/latest/index.html)
+- [JupyterHub documentation](https://jupyterhub.readthedocs.io/en/stable/)
+- [_The Turing Way_ Build a BinderHub Workshop](http://bit.ly/zero-to-binderhub-workshop)
 
 ## Definitions/glossary
 
 | | |
 |:---|:---|
-| Docker image or container | A set of instructions to create a computational environment.|
+| Docker image | A machine-readable set of instructions to create a soecified computational environment.|
```
Comment (sgibson91) [inline book/content/binderhub/binderhub.md]:
```suggestion
| Docker image | A machine-readable set of instructions to create a specified computational environment.|
```

**Comment by sgibson91** [comment]
Thank you @pherterich 💃

> Out of curiosity - is there a difference between a docker container and an image or is that just a preferred term?

I think so, yes. Here's my understanding:

- a Dockerfile is a set of human-readable instructions to docker
- a Docker image is a set of machine-readable instructions to docker
- a Docker container is the running environment

**Comment by sgibson91** [comment]
I just added that definition of docker container to the glossary.

**Comment by KirstieJane** [comment]
Amazing!

**Pull request merged**